### PR TITLE
added xsum_kernels_ascii utility

### PR DESCRIPTION
**Pull request by rmodrak**

### CODE REVIEW

**Comment by buildbot-princeton** [comment]
The files changed require a buildbot test. Buildbot has started.

**Comment by buildbot-princeton** [comment]
Your changes passed the buildbot test. See [build 125](http://specfem2d-buildbot.princeton.edu/builders/specfem2d_builder/builds/125).

File: src/postprocess/rules.mk [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+##
+sum_kernels_ascii_postprocess_OBJECTS = \
+	$O/sum_kernels_ascii.aux.o \
+	$(EMPTY_MACRO)
+
+sum_kernels_ascii_postprocess_SHARED_OBJECTS = \
+	$(EMPTY_MACRO)
+
+postprocess_OBJECTS += $(sum_kernels_ascii_auxiliaries_OBJECTS)
+postprocess_SHARED_OBJECTS += $(sum_kernels_ascii_auxiliaries_SHARED_OBJECTS)
```
Comment (QuLogic) [inline src/postprocess/rules.mk]: You've got a typo here. `auxiliaries` should be `postprocess`.

File: src/postprocess/rules.mk [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+####
+#### rules for executables
+####
+
+aux: $(postprocess_TARGETS)
+
+sum_kernels_ascii: xsum_kernels_ascii
+xsum_kernels_ascii: $E/xsum_kernels_ascii
+
+$E/xsum_kernels_ascii: $(sum_kernels_ascii_postprocess_OBJECTS) $(sum_kernels_ascii_auxiliaries_SHARED_OBJECTS)
```
Comment (QuLogic) [inline src/postprocess/rules.mk]: And here too.

File: src/postprocess/rules.mk [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+# data to be ensured and, more generally, to use and operate it in the
+# same conditions as regards security.
+#
+# The full text of the license is available in file "LICENSE".
+#
+#========================================================================
+
+## compilation directories
+S := ${S_TOP}/src/postprocess
+$(postprocess_OBJECTS): S := ${S_TOP}/src/auxiliaries
```
Comment (QuLogic) [inline src/postprocess/rules.mk]: And here.

File: src/postprocess/rules.mk [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+postprocess_SHARED_OBJECTS += $(sum_kernels_ascii_auxiliaries_SHARED_OBJECTS)
+
+
+#######################################
+
+####
+#### rules for executables
+####
+
+aux: $(postprocess_TARGETS)
```
Comment (QuLogic) [inline src/postprocess/rules.mk]: And probably here too.

Comment (rmodrak) [inline src/postprocess/rules.mk]: Looks like I left off a 'g' at the end of a sed command.  Surprisingly the
executable still managed to compile.  Let me close the current pull request
and submit a corrected one.

**Comment by komatits** [comment]
accept

On 03/06/2015 02:25 AM, buildbot-princeton wrote:
> Your changes passed the buildbot test. See build 125
> <http://specfem2d-buildbot.princeton.edu/builders/specfem2d_builder/builds/125>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geodynamics/specfem2d/pull/227#issuecomment-77489458>.
>

-- 
Dimitri Komatitsch
CNRS Research Director (DR CNRS), Laboratory of Mechanics and Acoustics,
UPR 7051, Marseille, France    http://komatitsch.free.fr

**Pull request closed without merging**